pkg/systemd/logs.js

```diff
@@ -28,6 +28,71 @@ $(function() {
     cockpit.translate();
     var _ = cockpit.gettext;
 
+    var problems_client = cockpit.dbus('org.freedesktop.problems');
```

Comment:
This should use `{ "superuser": "try" }` to match how journalctl is run.  If possible, this will then talk to the daemon as root and be able to see more problems.